packages/server/src/plugin/subscriptionCallback/index.ts

```diff
@@ -26,11 +25,19 @@ export function ApolloServerPluginSubscriptionCallback(
       const subscriptionExtension = request?.extensions?.subscription;
       // If it's not a callback subscription, ignore the request.
       if (!subscriptionExtension) return;
-      const {
-        callback_url: callbackUrl,
-        subscription_id: id,
+      let {
+        callbackUrl,
+        subscriptionId: id,
```

Comment:
for consistency with other fields I'd just keep it as `subscriptionId`
```suggestion
        subscriptionId,
```